aeron-client/src/main/c/util/aeron_fileutil.c

```diff
@@ -66,8 +68,7 @@ static int aeron_mmap(aeron_mapped_file_t *mapping, int fd, bool pre_touch)
 
     if (!hmap)
     {
-        AERON_SET_ERR(errno, "%s", "CreateFileMapping");
-        close(fd);
+        AERON_SET_ERR_WIN(GetLastError(), "%s", "CreateFileMapping");
```

Comment:
I like the use of the `AERON_SET_ERR_WIN` here I think that it is the right approach, but we need to make sure that the error handling is consistent on other OSs and that the call chain respects it.  We should do `AERON_SET_ERR` in the Linux/OSX version of aeron_mmap and all of the places that call this function (`aeron_map_existing_file`, `aeron_map_new_file`, `aeron_raw_log_map`, `aeron_raw_log_map_existing`) should be changed to use `AERON_APPEND_ERR` if the call to `aeron_mmap` fails.

Reply:
Done.